spec.md

```diff
@@ -441,7 +441,7 @@ be of interest.
 Each specification that defines how to serialize a CloudEvent will define how
 extension attributes will appear.
 
-Extension attribtue MUST be serialized using the same general pattern as all
+Extension attribute MUST be serialized using the same general pattern as all
```

Comment:
Isn't it plural?
attribute**s**

Reply:
this is why I ask for reviews!!  :-)  fixed